src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let count = 0;
+  const sum = days * 40;
+
+  if (days >= 3 && days < 7) {
+    count += sum - 20;
```

Comment:
I believe you don't need count variable now, cause you can return the result straight away

```suggestion
    return sum - 20;
```